scripts/deploy-manual.sh

```
@@ -0,0 +1,128 @@
+rsync -avz --delete \
+    --exclude='.git' \
+    --exclude='.github' \
+    --exclude='.env.local' \
+    --exclude='node_modules' \
+    --exclude='dist' \
+    --exclude='*.log' \
+    --exclude='backups' \
+    --exclude='.trae' \
+    --exclude='docs' \
+    --exclude='tests' \
+    $LOCAL_DIR/ $SERVER:$REMOTE_DIR/
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)